source/user-manual/registering/password-authorization-registration.rst

```diff
@@ -0,0 +1,196 @@
+.. Copyright (C) 2019 Wazuh, Inc.
+
+.. _password-authorization-registration:
+
+Registration service with password authorization
+================================================
+
+This registration method is similar to the :ref:`simple registration service <simple-registration-service>`, except that it allows additional protection of the Wazuh manager from unauthorized registrations by using a password.
+
```

Comment:
Please, explain the process. A setting needs to be enabled in the Wazuh manager, and then, you should be able to complete a registration using the password in the agent side.

Reply:
The users have 2 subtitles. They need to know that they must complete the 2 sections.

Reply:
Done but have doubts if it sounds good.

Reply:
I'd say provides rather than allows, but the rest sounds good to me